### ISSUE TITLE
WCAG 2.1 Viewer missing status messages

### ISSUE THREAD

**Issue by jolevesq** [fgpv-vpgf/fgpv-vpgf#3812]
There are no status messages to inform users that the data tables have been displayed on the page. Some users will not know that the page has been updated with the data table. This does not meet WCAG SC 4.1.3 Status Messages https://www.w3.org/WAI/WCAG21/Understanding/status-messages.html
Recommend using ARIA status role or live regions to indicate when the data table has finished loading
